models/subject.js

```diff
@@ -0,0 +1,27 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { LANG_LEVEL_ENUM }
+} = require('~/consts/validation')
+
+const subjectSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  price: {
```

Comment:
prolly should add validation so price can't have a negative value, aka min 0 or so

Reply:
fixed